README.md

```diff
@@ -1,36 +1,78 @@
 Piper
 =====
 
-GTK application to configure gaming mice. Piper uses libratbag via ratbagd.
+Piper is a GTK+ application to configure gaming mice, using libratbag via
+ratbagd.
+
+Piper is currently being redesigned. For the mockups, see the [Redesign
+Wiki][wiki-redesign].
```

Comment:
Not sure this sentence should be turned like that.

How about?:
For the design mockups, see the [Redesign Wiki][wiki-redesign].